traffic_portal/app/src/common/api/DeliveryServiceRequestService.js

```diff
@@ -26,6 +26,9 @@ var DeliveryServiceRequestService = function(Restangular, $http, $q, locationUti
 	this.createDeliveryServiceRequest = function(dsRequest) {
 		var request = $q.defer();
 
+		// strip out any falsy values or duplicates from consistentHashQueryParams
+		dsRequest.deliveryService.consistentHashQueryParams = _.compact(Array.from(new Set(dsRequest.deliveryService.consistentHashQueryParams)));
```

Comment:
this looks fine (pending testing), but I _would_ like to point something out

as long as we're using ES6 features like `Set`, it'd be nice to also use Array methods instead of underscore.

e.g.
```javascript
>> const test = ['test', null, 'quest', 'test'];
<- Array(4) [ "test", null, "quest", "test" ]
>> let result = _.compact(Array.from(new Set(test))); // instead of this
<- Array [ "test", "quest" ]
>> result = Array.from(new Set(test)).filter(i => i); //do this
<- Array [ "test", "quest" ]
```

Reply:
i thought we tried this and it fails to compile with browserify

`filter(i => i)`

Reply:
uhh, you might be right. I think browserify can't handle "fat arrow functions". In that case, you could do:
```javascript
Array.from(new Set(test)).filter(function(i){return i;})
```
which is much less pretty, but works just as well.